docs/security/GPG_KEY_ROTATION.md

```
@@ -0,0 +1,186 @@
+# GPG Release Key Rotation Playbook  
+**COINCUBE TECHNOLOGY LLC**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)